### PR TITLE
Remove player menu to streamline information

### PR DESCRIPTION
**Pull request by cailyncodes**

### CODE REVIEW

File: app/game/[gameId]/view/GameView.tsx [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+            <div className="flex flex-col justify-center items-center w-full mb-6">
+              {game.players.map((player, index) => {
+                if (player !== username) {
+                  return null
+                }
+                return (
+                  <div key={index} className="flex flex-row justify-center items-center w-full mb-4">
+                    <div className="max-w-3xl w-full font-mono text-sm flex flex-col">
+                      <pre className="whitespace-pre-line">
+                        {game[username]}
```
Comment (5720axolotls) [inline app/game/[gameId]/view/GameView.tsx]: It's kinda weird to me that this mapping of player to info isn't some field inside of game, but not a new change in this pr. It's just that the game might end up getting more complicated down the road.

File: app/game/[gameId]/view/GameView.tsx [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+          <div className="max-w-lg w-full h-full flex flex-col self-center items-center justify-center mb-6 text-lg">
+            <h2>Game Code: {gameCode}</h2>
+          </div>
+          <div className="max-w-lg w-full h-full flex flex-col self-center items-center justify-center">
+            <div className="flex flex-row justify-center items-center w-full mb-6">
+              <p className='text-md font-bold'>{game.start}</p>
+            </div>
+            <div className="flex flex-col justify-center items-center w-full mb-6">
+              {game.players.map((player, index) => {
+                if (player !== username) {
```
Comment (5720axolotls) [inline app/game/[gameId]/view/GameView.tsx]: Why is it necessary to map like this? Can't you just index into game[username] directly?

File: app/game/[gameId]/view/GameView.tsx [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+                  </pre>
+                </div>
+              </div>
+            </div>
+          ) : (
+            <div className="max-w-lg w-full h-full flex flex-col self-center items-center justify-center mb-6">
+              <button
+                className="bg-red-500 hover:bg-red-700 text-white font-bold py-2 px-4 rounded w-2/5"
+                onClick={() => {
+                  if (window.confirm("Are you sure? Cancel to close.")) {
```
Comment (5720axolotls) [inline app/game/[gameId]/view/GameView.tsx:63]: Oh that's nice.

**Review by 5720axolotls — APPROVED** [review]

File: app/game/[gameId]/view/GameView.tsx (hunk 2 above)
Comment (cailyncodes) [reply app/game/[gameId]/view/GameView.tsx]: Omg lmao it is so not necessary

**Pull request merged**